lib/resources/datav2/websockets_v2.js

```diff
@@ -178,13 +186,17 @@ class AlpacaStreamV2Client extends events.EventEmitter {
     if (
       this.session.subscriptions.trades.length > 0 ||
       this.session.subscriptions.quotes.length > 0 ||
-      this.session.subscriptions.bars.length > 0
+      this.session.subscriptions.bars.length > 0 ||
+      this.session.subscriptions.dailyBars.length > 0 ||
+      this.session.subscriptions.statuses.length > 0
```

Comment:
not a big deal but this could be cleaned up a bit
```js
const {trades, quotes, bars, dailyBars, statuses } = this.session.subscriptions
if (trades.length ||  quotes.length || bars.length || dailyBars.length || statuses.length) {

}
```